lib/skate_web/controllers/auth_controller.ex

```diff
@@ -71,4 +71,45 @@ defmodule SkateWeb.AuthController do
   def callback(%{assigns: %{ueberauth_failure: %{provider: :keycloak}}} = conn, _params) do
     send_resp(conn, :unauthorized, "unauthenticated")
   end
+
+  # https://github.com/mbta/arrow/blob/372c279e04866509f1e287e07844d61cc243850b/lib/arrow_web/controllers/auth_controller.ex#L57-L66
+  defp sign_out_url(conn, auth) do
+    case UeberauthOidcc.initiate_logout_url(
+           auth,
+           %{
+             post_logout_redirect_uri: url(conn, ~p"/")
+           }
+         ) do
+      {:ok, url} ->
+        url
+
+      _ ->
+        nil
+    end
+  end
+
+  @spec logout(Plug.Conn.t(), map()) :: Plug.Conn.t()
+  def logout(conn, %{"provider" => "keycloak"}) do
+    case Guardian.Plug.current_claims(conn) do
+      %{"sign_out_url" => sign_out_url} when not is_nil(sign_out_url) ->
+        conn
+        |> session_cleanup()
+        |> redirect(external: sign_out_url)
+
+      # The router makes sure we can't call `/auth/:provider/keycloak`
```

Comment:
**nitpick:** I think there's a typo in the path, is that supposed to be `/auth/:provider/callback`?

Reply:
Yep! thanks!
```suggestion
      # The router makes sure we can't call `/auth/:provider/callback`
```